batch/batch/batch.py

```diff
@@ -95,38 +143,44 @@ def job_record_to_dict(record: Dict[str, Any], name: Optional[str]) -> JobListEn
     if cost_breakdown is not None:
         cost_breakdown = cost_breakdown_to_dict(json.loads(cost_breakdown))
 
-    return {
-        'batch_id': record['batch_id'],
-        'job_id': record['job_id'],
-        'name': name,
-        'user': record['user'],
-        'billing_project': record['billing_project'],
-        'state': record['state'],
-        'exit_code': exit_code,
-        'duration': duration,
-        'cost': coalesce(record.get('cost'), 0),
-        'msec_mcpu': record['msec_mcpu'],
-        'cost_breakdown': cost_breakdown,
-    }
-
-
-async def cancel_batch_in_db(db, batch_id):
+    return cast(
+        JobListEntryV1Alpha,
+        {
+            'batch_id': record['batch_id'],
+            'job_id': record['job_id'],
+            'name': name,
+            'user': record['user'],
+            'billing_project': record['billing_project'],
+            'state': record['state'],
+            'exit_code': exit_code,
+            'duration': duration,
+            'cost': coalesce(record.get('cost'), 0),
+            'msec_mcpu': record['msec_mcpu'],
+            'cost_breakdown': cost_breakdown,
```

Comment:
Conflict msg, I have meged incoming with our version.

++<<<<<<< HEAD
 +            'cost_breakdown': cost_breakdown,
++||||||| 6a6c38d5a
++            'cost_breakdown': record['cost_breakdown'],
++=======
+             'cost_breakdown': record['cost_breakdown'],
+             'always_run': bool(record['always_run']),
+             'display_state': None,
++>>>>>>> bea04d9

Reply:
Good one — a nice straight-forward one to resolve